contents/blog/eu-cloud-hosting.md

```diff
@@ -0,0 +1,89 @@
+---
+date: 2022-10-17
+title: How we built EU Cloud hosting, from idea to launch
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Inside PostHog", "Startups"]
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/posthog-eu-blog.png
+featuredImageType: full
+---
+
+We recently launched EU Cloud, providing all the benefits of cloud-hosted PostHog with data staying in the EU. We thought it would be interesting to go over why and how we built EU Cloud hosting, from idea to launch.
```

Comment:
I'll tag @andyvan-ph so we can have consensus and consistency on this, but we should make a clear decision on `Cloud EU` and `EU Cloud`. 

My vote is that we should take the `/eu` page as a point of standardization and call the _product_ `PostHog Cloud EU`. If that's the product name then it is fine to say that we have launch _an_ EU Cloud or that `PostHog now has an EU Cloud offering` - but we should change this to...

```suggestion
We recently launched PostHog Cloud EU, providing all the benefits of cloud-hosted PostHog with data staying in the EU. We thought it would be interesting to go over why and how we built Cloud EU hosting, from idea to launch.
```

Reply:
EU Cloud is the North American way to say it. Cloud EU is the European way to say it. This probably means we should choose the latter.